src/notification/remote-push-notification/RemotePushNotificationTask.ts

```diff
@@ -173,6 +173,11 @@ export default class RemotePushNotificationTask implements NotificationTask {
     return Promise.resolve();
   }
 
+  public async sendEndUserErrorNotification(data: EndUserErrorNotification, user: User, tenant: Tenant, severity: NotificationSeverity): Promise<void> {
```

Comment:
I would expect that we also send a remote push notif

Reply:
Tested in debugger.  
I got the app but the mobileToken for my user will be updated in next db backup for local env. I guess.

Reply:
> Tested in debugger.
> I got the app but the mobileToken for my user will be updated in next db backup for local env. I guess.

Right!
I'll send it to you by email ;-)

Reply:
Sounds good :)